index.html

```diff
@@ -0,0 +1,162 @@
+    crossorigin="anonymous"></script>
+  <script src="./src/semantic/semantic.min.js"></script>
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    .balance {
+      font-family: monospace;
+    }
+
+    h2 {
+      background-color: lightgrey;
+      display: inline-block;
+      padding: 5px 20px;
+      border-radius: 10px;
+    }
+
+    .fn-container {
+      display: flex;
+      align-items: stretch;
+      height: 450px;
+      justify-content: space-between;
+      width: 100%;
+
+    }
+
+    .fn-container>div {
+      flex: 0 0 30%;
+    }
+
+    .fn-container .segment {
+      margin-top: 0;
+      margin-bottom: 0;
+
+    }
+
+    .fn-container button {
+      float: right;
+    }
+
+    .create-wallet {
+      position: absolute;
+      right: 0;
+      top: 50%;
+      transform: translateY(-50%);
+    }
+
+    button.ui.primary.button {
+      background-color: #1a56f0;
+    }
+
+    button.ui.primary.button:hover {
+      background-color: #1a56f0;
+    }
+
+    button.ui.primary.button:focus {
+      background-color: #1a56f0;
+    }
+  </style>
+</head>
+<title>js-kit-demo</title>
+<body>
+  <div class="ui stackable grid container" style='padding: 50px 0;'>
+    <div class='row'>
+      <img src='./images/OmiseGO_Logo.svg' />
+      <button class="ui primary button create-wallet" onclick="newWallet()">Create new Wallet</button>
+    </div>
+    <div id="walletArea" class="wallet-area ui segment" style='width: 100%'>
+      <div class='row'>
+        <button class="ui primary button" onclick="restoreSeed()">Restore from seed</button>
+        <div class='ui input'>
+          <input type="text" id="seed" value="" size="80" placeholder='your seed phase...'></input>
+        </div>
+      </div>
+      <div>
+      </div>
+    </div>
+    <div class='row'>
+      <div id="addressArea" class="address-area ui segment" style="display:none; width: 100%;">
+        <h2>Addresses</h2>
+        <div>
+          <button class="ui primary button" onclick="newAddress('')">Generate new address</button>
+          <button class="ui primary button" onclick="showBalances()">Refresh balances</button>
+        </div>
+        <h4>Rootchain balance</h4>
+        <div id="rootchainBalance" class="balance"></div>
+        <h4>Childchain balance</h4>
+        <div id="childchainBalance" class="balance"></div>
+      </div>
+    </div>
+    <div class='row'>
+      <div id="functionArea" class="function-area grid container" style="display:none; width: 100%;">
+        <div class='fn-container'>
+          <div class='ui segment'>
+            <h2>Deposit</h2>
+            <form class="ui form">
+              <div class="field">
+                <label>From address:</label>
+                <input type="text" id="depositFromAddress"></input>
+              </div>
+              <div class="field">
+                <label>Token contract address (blank for ETH):</label>
```

Comment:
could you change this to Wei ? (default unit is in Wei). Some people were mislead in the past thinking that they are transferring 1 ETH.